src/generator/pages.clj

```diff
@@ -24,60 +24,104 @@
+
 
 (defn render-page-footer
   "Must be called from hiccup. Outputs common page footer in a vector."
   []
-  [:footer {:class "footer"}
-   [:span "&copy 2023 Cartman Digital Oy"]
-   [:div.social-media
-    (link-to "https://www.linkedin.com/company/cartmandigital/"
-             [:span.icon
-              [:svg {:xmlns "http://www.w3.org/2000/svg" :width "24" :height "24" :viewBox "0 0 24 24"}
-               [:path {:fill "currentColor" :d "M19 0h-14c-2.761 0-5 2.239-5 5v14c0 2.761 2.239 5 5 5h14c2.762 0 5-2.239 5-5v-14c0-2.761-2.238-5-5-5zm-11 19h-3v-11h3v11zm-1.5-12.268c-.966 0-1.75-.79-1.75-1.764s.784-1.764 1.75-1.764 1.75.79 1.75 1.764-.783 1.764-1.75 1.764zm13.5 12.268h-3v-5.604c0-3.368-4-3.113-4 0v5.604h-3v-11h3v1.765c1.396-2.586 7-2.777 7 2.476v6.759z"}]]
-              [:span {:class "sr-only"} "LinkedIn"]])
-    
-    (link-to "https://twitter.com/CartmanDigital"
-             [:span.icon
-              [:svg {:xmlns "http://www.w3.org/2000/svg" :width "24" :height "24" :viewBox "0 0 1200 1227"}
-               [:path {:fill "rgb(255,255,255)" :d "M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"}]]
-              [:span {:class "sr-only"} "Twitter"]])
-    
-    (link-to "https://www.itewiki.fi/cartman-digital"
-             [:span.icon
-                 [:svg {:xmlns "http://www.w3.org/2000/svg"  :width "24" :height "24" :viewBox "0 0 177.7 177.7" :x "0px" :y "0px" :style "enable-background:new 0 0 177.7 177.7;" :xml:space "preserve"}
-                  [:g 
-                   [:path {:fill "rgb(255,255,255)"  :d "M84.3,129l-40-40.3L0.2,44.2L0,133.5l39.8,39.6L84.3,129L84.3,129z M93.4,48.7l40,40.3l44.1,44.4l0.2-0.2V44.2L137.9,4.6L93.4,48.7z"}]
-                   [:polygon  {:fill "rgb(255,255,255)"   :points "129,93.4 88.6,133.4 44.2,177.5 133.5,177.7 173.1,137.9"}]
-                   [:polygon  {:fill "rgb(255,255,255)"  :points "48.7,84.3 89,44.3 133.5,0.2 44.2,0 4.6,39.8"}]]]
-                 [:span {:class "sr-only"} "Itewiki"]])]])
+  (let [cta-button-text "CONTACT US"
+        cta-button-url "https://example.com/more-info"]
+    [:div.place-items-center.grid.grid-cols-1
+     [:div.foot.grid.sm:justify-items-center.grid-cols-1.sm:grid-cols-3 {:class "footer"}
+      (menu-footer)
+      [:hr]
+      [:ul
+       [:li "Social media"]
+       [:li
+        [:a
+         {:href "https://twitter.com/CartmanDigital"}
+         twitter-icon
+         "LinkedIn"]] 
```

Comment:
Twitter link and icon with linked in text? Could you replace the link and icon with Linkedin icon and link to cartman digital on linkedin?